### ISSUE TITLE
Horreum: Upload all available run into Horreum and Implement daily Jenkins job

### ISSUE THREAD

**Issue by andyuk1986** [keycloak/keycloak-benchmark#712]
The next step for the Horreum reports would be:

- upload the existing runs (available in Github) into Horreum;
- create Jenkins job to run daily and upload the generated benchmark JSON to Horreum; 

Please find below the URL to the Test Run config in Horreum: https://horreum.corp.redhat.com/test/308
The URL to change detections page for scalability benchmark is: https://horreum.corp.redhat.com/changes?test=keycloak-benchmark

**Comment by andyuk1986** [comment]
1) So, all existing runs were uploaded to Horreum. The full data (including gatling statistics) contains the reports starting from Jan 26, 2024. The data before contains only stats calculated from Prometheus. So I have updated the existing reports before Jan 26 also to include the environment information (i.e. context{} JSON element). Also I have found out when we have started to run MultiAZ deployemtn with ISPN CrossDC and reports generated before marked as ran with ISPN single deployment. 

2) Job is created in Keycloak Jenkins which is scheduled to work daily ~13:00CET, download the latest report JSON file and upload it to Horreum. Also the job is implemented to be parameterized, i.e. it takes date (in case some day was missed) and uploads data starting from that date. The PR for the job is sent here: https://gitlab.cee.redhat.com/keycloak/keycloak-qe-jenkins-csb/-/merge_requests/161
The job itself is located here: https://master-jenkins-csb-keycloak-qe.apps.ocp-c1.prod.psi.redhat.com/view/Performance/job/kb-horreum-upload/
